proposals/incubation/strimzi.md

```diff
@@ -0,0 +1,134 @@
+# Strimzi Incubation Stage Proposal
+
+## Introduction
+
+Strimzi is a project focused on running [Apache Kafka][ak] on [Kubernetes][kube].
+Apache Kafka is the _de facto_ standard platform for building real-time data pipelines.
+It provides support for high-throughput / low-latency messaging, as well as sophisticated development options that cover all the stages of a distributed data streaming pipeline, from ingestion, through storage, and on to processing.
+But running Kafka on Kubernetes can be complex and tedious.
+Strimzi makes it easy by using the Kubernetes operator pattern and declarative deployments.
+It provides operators not just for deploying Kafka clusters, but also for managing topics, users and connectors to make sure running Apache Kafka feels Kube-native.
+
+Strimzi also integrates with many other CNCF projects such as:
+
+* Kubernetes
+* [Prometheus][prom] for metrics and monitoring
+* [Open Policy Agent][opa] for authorization
+* [KEDA][keda] for autoscaling of Kafka clients
+* [Jaeger][jaeger] and [OpenTelemetry][otel] for tracing
+
+In addition, Strimzi is used by [Knative][knative] to provide its Kafka source.
+
+Strimzi can be installed using [Helm][helm] charts or using [OperatorHub.io][operatorhub], which is part of the [Operator Framework][operatorframework] project.
```

Comment:
If I remember correctly, OperatorHub is NOT part of the Operator Framework project. When the Operator Framework was given to the CNCF, OperatorHub was not part of it and has it's own GitHub org. I think this needs to be updated.

Reply:
I guess we can maybe rephrase it something like this?

```suggestion
Strimzi can be installed using [Helm][helm] charts or using Operator Lifecycle Manager, which is part of the [Operator Framework][operatorframework] project.
Both the Helm charts and OLM bundles are available on [Artifact Hub](https://artifacthub.io/).
```

The Operator Lifecycle Manager seems to be part of Operator Framework (https://github.com/operator-framework/operator-lifecycle-manager) -> just the Operator directory isn't. But the artifacts are listed also in Artifact Hub that is part of CNCF if I remember correctly.

Reply:
I agree with Jakub's proposal. @mattfarina let me know what do you think about it, and I will update the PR. Thanks!

Reply:
I pushed the suggested change, let me know if that's not ok and we can re-open the conversation.